app/views/godmin/filters/_boolean.html.erb

```diff
@@ -0,0 +1,7 @@
+<%
+# Local variables:
+#   `form` is the filter form object.
+#   `filter` is the filter being rendered
+%>
+
+<%= form.select_filter_field(filter) %>
```

Comment:
I think we should keep the partial for boolean and not override `to_partial_path`. Otherwise, the user cannot override a boolean filter view.